### DISCUSSION TITLE
Question about Licenses

### DISCUSSION THREAD

**Discussion by 34j** [coqui-ai/TTS#2907] [General Q&A]
I believe the code in this repository is based on numerous other open source projects, but the license on PyPI and GitHub is MPL 2.0 only. Does this mean the maintainers have specifically asked the author of each project for permission to release it under MPL 2.0? 
(I am not trying to attack the maintainers, I am genuinely asking the question. Sorry if I sounded offensive.)

**Comment by erogol** [comment]
There is no project that is in conflict with MPL as far as we are aware. Q14 here https://www.mozilla.org/en-US/MPL/2.0/FAQ/
